server/session/world.go

```diff
@@ -744,6 +744,17 @@ func (s *Session) ViewEmote(player world.Entity, emote uuid.UUID) {
 	})
 }
 
+// ViewWorldSpawn ...
+func (s *Session) ViewWorldSpawn(pos cube.Pos) {
+	blockPos := protocol.BlockPos{int32(pos[0]), int32(pos[1]), int32(pos[2])}
+	s.writePacket(&packet.SetSpawnPosition{
+		SpawnType:     packet.SpawnTypeWorld,
+		Position:      blockPos,
+		Dimension:     1,
```

Comment:
I think 1 is the nether, not sure if that's right? Maybe this works too though, I don't know.

Reply:
You are right, I was looking at PM for this but I looked at the wrong file, woops 😅